recipes/certs.rb

```diff
@@ -0,0 +1,20 @@
+bash 'materialize_truststores from hdfs' do
+  user "root"
+  code <<-EOH
+        su #{node['hops']['user']}
+        #{node['hops']['base_dir']}/bin/hdfs dfs copyToLocal "/user/#{node['hadoop_spark']['user']}/cacerts.jks" "#{node['hadoop_spark']['conf_dir']}/"
+        chmod 444 "#{node['hadoop_spark']['conf_dir']}/cacerts.jks"
+        #{node['hops']['base_dir']}/bin/hdfs dfs copyToLocal "/user/#{node['hadoop_spark']['user']}/cacerts.pem" "#{node['hadoop_spark']['conf_dir']}/" 
+        chmod 444 "#{node['hadoop_spark']['conf_dir']}/cacerts.pem"
+  EOH
+end
+
+hops_hdfs_directory "/user/#{node['hadoop_spark']['user']}/cacerts.jks" do
```

Comment:
When one node reaches this block, it will delete the files from HDFS. How will the rest of the machines localize them?

Reply:
@kouzant You are right! How would we clean up the file from hdfs then? We could also leave it there, doesn't hurt I guess

Reply:
There is no direct way. There has to be a recipe that must run after hadoop_spark::yarn has ran globally and delete the source files. Yes, we could leave them there as long as all `copyFromLocal` operations use `-f` flag to overwrite them.